src/components/IdentityVerificationProcess.tsx

```diff
@@ -1,25 +1,50 @@
-import { useState } from 'react';
+import { useEffect, useMemo, useRef, useState } from 'react';
 import Header from './Header';
 import ProgressBar from './ProgressBar';
 import IdentityForm from './IdentityForm';
 import ChallengeVerification from './ChallengeVerification';
 import CompletionPage from './CompletionPage';
 import { useSnapshot } from 'valtio';
 import { appState } from '~/App';
-import { SignerProvider } from '@reactive-dot/react';
+import { getSs58AddressInfo } from 'polkadot-api';
+import { IdentityVerificationStates } from '~/constants';
 
+enum Stages {
+  SetIdentityForm = 0,
+  Challenges = 1,
+  Congrats = 2,
+}
```

Comment:
improve state namings for stages
```
enum RegistrationStages {
  Submission = 0,
  Verification = 1,
  Complete = 2,  
```

Reply:
Solved in 5f4b490f251205a34b8049f805dc8bf6dc16d652